application/tests/unit/test_handler.py

```diff
@@ -0,0 +1,44 @@
+import os
+import json
+import pytest
+import application.handler as handler
+
+
+class TestHandler:
+
+    log_path: str = "tests/unit/test_log/rd.log"
+    json_path: str = "tests/unit/test_json/"
+    os.environ["LOGGER"] = "Test"
+
+    def test_valid_ccs_request(self):
+        self.purge_test_logs()
+        request = self.fetch_json("dos_road_distance_api_happy.json")
+        response = handler.process_road_distance_request(request, None)
+        assert response["status_code"] == 200
+        assert os.path.isfile(self.log_path)
+
+    def test_invalid_ccs_request(self):
+        self.purge_test_logs()
+        request = self.fetch_json("dos_road_distance_api_invalid_missing_element.json")
+        response = handler.process_road_distance_request(request, None)
+        assert response["status_code"] == 500
+        assert os.path.isfile(self.log_path)
+
+    def test_invalid_log_name_raises_value_error(self):
+        self.purge_test_logs()
+        os.environ["LOGGER"] = "DoesNotExist"
+        request = self.fetch_json("dos_road_distance_api_invalid_missing_element.json")
+        response = handler.process_road_distance_request(request, None)
+        assert response["status_code"] == 500
+        assert os.path.isfile(self.log_path) == False
+
+    def purge_test_logs(self):
+        if os.path.isfile(self.log_path):
+            os.remove(self.log_path)
+
+    def fetch_json(self, file_name: str) -> str:
```

Comment:
I think we have this code in 3 places now - perhaps time for a helper class

Reply:
I thought about this, something we can add in on any ticket, but this one is doing something slightly different (not that it couldn't be coded around)